setup.py

```diff
@@ -55,6 +55,8 @@
 
 # Note: cocotb is not installed properly yet and is missing dependencies and binaries
 # We can still import other files next to setup.py, as long as they're in MANIFEST.in
+# The below line is necessary for PEP517 support (gh-2091)
+sys.path.append(path.dirname(__file__))
```

Comment:
This is needs because we edited pyproject.toml, or was it already needed before?

Reply:
Only because we edited pyproject.toml. Basically we changed build backends.

Having a pyproject.toml enabled PEP517 support, but if you don't specify a backend it defaults to the legacy backend (behaves like `python setup.py bdist_wheel`), the new backend `setuptools.build_meta` doesn't respect MANIFEST.in since it isn't a part of the PEP517 spec, it's a part of building an sdist. Eventually the legacy backend will be removed and all projects will be required to be fully PEP517 compliant. Specifying the legacy backend via `setuptools.build_meta:__legacy__` is already deprecated.

Reply:
It would be good if you could capture that in the commit message. Supporting PEP517 is probably also release note worthy too.

I'd remove the PR reference from this comment, because the PR title and description doesn't help at all.